source/Uniform/UniformDatabase.cs

```diff
@@ -62,7 +61,7 @@ public void EnterInMemoryMode()
             _metadata.Databases = newDatabases;
         }
 
-        public void LeaveInMemoryMode(bool flush = false)
+        public void LeaveInMemoryMode(bool flush = false, bool dropBeforeFlush = true)

```

Comment:
You are breaking backward compatibility with this code. Don't use signature of existing public methods. Use overloaded method instead.